src/ducks/balance/components/AccountsImporting.styl

```diff
@@ -0,0 +1,24 @@
+@require 'settings/breakpoints'
+
+$header-margin-bottom = -28px
+
+.content
+    height 245px
+    color white
```

Comment:
Can you use a CSS variable here? Not necesarily a new one, but something like `primaryContrastTextColor`?